docs/tables/aws_securityhub_product.md

```diff
@@ -0,0 +1,46 @@
+# Table: aws_securityhub_product
+
+Security hub Products provides information about product integrations in Security Hub.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  product_arn,
+  company_name,
+  description
+from
+  aws_securityhub_product;
+```
+
+
+### List products provided by AWS
```

Comment:
Are we able to get products created by the account/users instead? These may be more useful in a query (and produce a more manageable number of results as well).

Reply:
@cbruno10 I checked with the entire dataset and with @rajeshbal65 , I guess the good example for this as provided list by company. Here we have provided by AWS. There is no individual user related information available. Some product ARN has account ID information. We can filter on these, but most will be searching based the 3rd party vendor name.